README.md

```diff
@@ -1 +1,45 @@
-# tic-tac-toe
\ No newline at end of file
+# Tic-Tac-Toe
+> This is an implementation the Tic-Tac-Toe gane in Ruby, in addition, it is going to be implemented in 4 phases:
+- Project setup
+- User interface
+- Game logic
+- Game instructions and final touches
```

Comment:
You can include all the information related to how to play, and adjust it later if necessary. This will also serve as a good way to plan in advance.